tests/e2e_test/module_deletion_test.go

```diff
@@ -0,0 +1,187 @@
+package e2e_test
+
+import (
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	v2 "github.com/kyma-project/lifecycle-manager/internal/declarative/v2"
+	"github.com/kyma-project/lifecycle-manager/pkg/testutils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+var _ = Describe("Non Blocking Module Deletion", Ordered, func() {
+	kyma := testutils.NewKymaWithSyncLabel("kyma-sample", "kcp-system", "regular",
+		v1beta2.SyncStrategyLocalSecret)
+	GinkgoWriter.Printf("kyma before create %v\n", kyma)
+
+	BeforeAll(func() {
```

Comment:
This `beforeall` is not need, if KymaCRD is not installed, if it happens, in later `Eventually` when you create kyma CR it will fail and continue trying to create.

I already refactoring the shared BeforeAll and AfterAll statements in this [PR](https://github.com/kyma-project/lifecycle-manager/pull/911/files#diff-996d9d42109a1ee29b73f16aaea9f398b2a2a374f62f47b6ba881f845be21c57R16-R17), so please do not introduce more in.